sphinxgallery/gen_rst.py

```diff
@@ -308,9 +310,10 @@ def generate_dir_rst(directory, fhindex, examples_dir, gallery_dir, gallery_conf
     """)  # clear at the end of the section
 
 
-def make_thumbnail(in_fname, out_fname, width, height):
-    """Make a thumbnail with the same aspect ratio centered in an
-       image with a given width and height
+def scale_image(in_fname, out_fname, max_width, max_height):
+    """Scales an image with the same aspect ratio centered in an
+       image with a given max_width and max_height
+       if in_fname == out_fname the image can only be scaled down
```

Comment:
Minor style comment: the short description of a function is supposed to fit on one line, see PEP 257 and/or [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) convention.